### DISCUSSION TITLE
CSS Modules

### DISCUSSION THREAD

**Discussion by jimbits** [honojs/hono#3258] [Q&A]
I am exploring Hono and wanted to use css modules with the build in JSX library. I have been searching the docs and discussion on how to do that. Is it possible or would I have to write a library similar to hono/css that would do this.

Thanks for any feedback.

Jim

**Comment by mybigman** [comment]
I too looking to use css modules with hono/honox... as I dont use a frontend and use hono to render server side.

Did you find a solution?
